diagrammer/scene/basic.py

```diff
@@ -331,17 +331,27 @@ def svg(self) -> str:
 
 
 class Arrow(SceneObject):
+    # Type aliases
+    Path = str
+
     HEAD = 'head'
     TAIL = 'tail'
+    BEZIER_RADIANS = math.radians(30)
+
+    STRAIGHT = 'straight'
```

Comment:
you could add the path settings to ArrowSettings instead maybe

Reply:
that's what I did at first lol but there was some problem with it

Reply:
oh yeah the problem is I have a class variable called smth like PY_REFERENCE_ARROW_SETTINGS that all PyReferences are initialized with, so if I modify it it would modify the settings of all PyReferences.

Reply:
and it wasn't good design to allow the user to modify the settings. you could have "get_settings()" function in Arrow and have the user modify that directly, but the problem is when you modify certain settings you need to reset the cache. Thus, you need a "set_settings()" function, but users (as in us) might forget to call set_settings() and just do get_settings() directly. Another solution is to have a set_path() function and no get_settings() function, but that means Arrow has to know about the details of ArrowSettings, which didn't seem right. Overall, it seems like the way we designed settings isn't meant for them to be modified after creation